src/controllers/trainee/routes.ts

```diff
@@ -3,14 +3,283 @@ import TraineeController from './Controller';
 import authMiddleWare from '../../libs/routes/authMiddleWare';
 import validationHandler from '../../libs/routes/validationHandler';
 import { validation } from '../user/validation';
+import swaggerOptions from '../../libs/swagger';
+import * as swaggerUi from 'swagger-ui-express';
 
 const traineeRouter: Router = Router();
-traineeRouter
 
-.get('/', authMiddleWare('getUsers','read'), validationHandler(validation.get),TraineeController.getAll)
-.get('/:id', authMiddleWare('getUsers','read'), validationHandler(validation.get), TraineeController.getById)
-.post('/', authMiddleWare('getUsers','read'), validationHandler(validation.create), TraineeController.create)
-.put('/:id', authMiddleWare('getUsers','read'), validationHandler(validation.update), TraineeController.update)
-.delete('/:id', authMiddleWare('getUsers','read'), validationHandler(validation.delete), TraineeController.delete);
+traineeRouter.use('/api', swaggerUi.serve, swaggerUi.setup(swaggerOptions));
```

Comment:
There should be a single route for the swagger doc.
i.e for user and trainee docs.

Reply:
Done